templates/_includes/postslist.njk

```diff
@@ -1,6 +1,6 @@
 <ol reversed start="{{ postslist.length + 1 }}" class="postlist">
 {% for post in postslist | reverse %}
-  <li value={{ loop.index | reverseAndPad(postslist.length) }} class="postlist-item">
+  <li value={{  loop.revindex | minimumTwoDigits }} class="postlist-item">
```

Comment:
4️⃣  Nunjucks provides `loop.revindex`, which decrements so the filter doesn't have to. Now that the filter has a more specific job, I figured a name change was in order.

Reply:
ooooh clever